app/queries/my_facilities/blood_pressure_control_query.rb

```diff
@@ -53,24 +53,27 @@ def cohort_missed_visits_count_by_facility
 
   def overall_patients
     @overall_patients ||= Patient
-                          .where('recorded_at < ?', Time.current.beginning_of_day - REGISTRATION_BUFFER)
-                          .where(registration_facility: facilities)
+                            .hypertension_only
+                            .where('recorded_at < ?', Time.current.beginning_of_day - REGISTRATION_BUFFER)
+                            .where(registration_facility: facilities)
   end
 
   def overall_controlled_bps
     @overall_controlled_bps ||=
       LatestBloodPressuresPerPatient
```

Comment:
PS, what's our plan for these views? Do we plan to filter the results returned by them, or filter the data that goes into them in the first place?

Reply:
So from my understanding so far, only 2 view-implementations actually need to change, the rest can be done using scopes alone:

* The Inactive facilities (populated by `MyFacilities::OverviewQuery`)

This is because this view doesn't join on `Patient` at all. If we joined with patients, we can scope this query up in application code.

*  `MyFacilities::RegistrationsQuery` (period-wise registrations)

This is a weird one. This does join on patients but returns an aggregate. So we might have to stick the htn-only clause in the view for this to work correctly.